File: Leet Code/1450_number_of_students_doing_home_work_at_a_given_time.cpp
class Solution {
public:
    int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime) {
        int arr[1002];
        for(int i = 0; i < 1002; i++){
            arr[i] = 0;
        }
        for(int i = 0; i < startTime.size(); i++){
            arr[startTime[i]] += 1;
        }
        for(int i = 0; i < endTime.size(); i++){
            arr[endTime[i] + 1] -= 1;
        }
        for(int i = 1; i < 1002; i++){
            arr[i] += arr[i-1];
        }
        return arr[queryTime];
    }
};

File: Leet Code/1450_number_of_students_doing_home_work_at_a_given_time.py
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        arr = [0]*1002
        for i in startTime:
            arr[i] += 1 
        for i in endTime:
            arr[i+1] -= 1
        for i in range(1,1002):
            arr[i] += arr[i-1]
        return arr[queryTime]
        
